### PR TITLE
Feat/add stocks dataset example

### PR DESCRIPTION
**Pull request by waelsy123**
# Rationale for this change

- More test example in the repo provide better documentation

# What changes are included in this PR?

- stock market dataset
- script to run the test

# Are these changes tested?

Yes

### CODE REVIEW

**Comment by Dustin-Ray** [comment]
Hi @waelsy123 would you mind resolving the merge conflict real quick and then we can go ahead and merge? Thanks

File: .github/workflows/lint-and-test.yml [hunk 1]
```diff
@@ -130,6 +130,8 @@ jobs:
         run: cargo run --example plastics
       - name: Run sushi example
         run: cargo run --example sushi
+      - name: Run stocks example
+        run: cargo run --example stocks
```
Comment (Dustin-Ray) [inline .github/workflows/lint-and-test.yml:134]: This change is no longer needed and can safely be reverted

Comment (waelsy123) [reply .github/workflows/lint-and-test.yml]: reverted

**Comment by waelsy123** [comment]
> Hi @waelsy123 would you mind resolving the merge conflict real quick and then we can go ahead and merge? Thanks

conflicts resolved

**Review by Dustin-Ray — APPROVED** [review]
LGTM

**Pull request merged**